Cargo.toml

```diff
@@ -19,20 +20,22 @@ publish = false
 
 [workspace.dependencies]
 # reth
-reth = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
-reth-chainspec = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
+reth = { git = "https://github.com/paradigmxyz/reth", features = ["optimism"], version = "1.0.4" }
+reth-chainspec = { git = "https://github.com/paradigmxyz/reth", features = ["optimism"], version = "1.0.4" }
 reth-discv5 = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
 reth-execution-errors = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
-reth-execution-types = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
+reth-execution-types = { git = "https://github.com/paradigmxyz/reth", features = ["optimism"], version = "1.0.4" }
 reth-exex = { git = "https://github.com/paradigmxyz/reth", features = ["serde"], version = "1.0.4" }
 reth-network-peers = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
 reth-node-api = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
 reth-node-ethereum = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
-reth-primitives = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
-reth-provider = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
-reth-revm = { git = "https://github.com/paradigmxyz/reth", version = "1.0.4" }
+reth-primitives = { git = "https://github.com/paradigmxyz/reth", features = ["optimism"], version = "1.0.4" }
+reth-provider = { git = "https://github.com/paradigmxyz/reth", features = ["optimism"], version = "1.0.4" }
+reth-revm = { git = "https://github.com/paradigmxyz/reth", features = ["optimism"], version = "1.0.4" }
```

Comment:
the `features = ["optimism"]` in this file should be enabled in the `daimo-names/Cargo.toml` instead